Automattic/woocommerce-payments
# Initial start of settings for multi-currency

Fixes #1768

#### Changes proposed in this Pull Request

* Added Multi-Currency tab under WooCommerce > Settings
* Sub tabs are Currencies and Store settings.
* Enabled currencies lists allows to add and remove currencies, and also provides link to edit a single currency.
* Single currency page allows setting the exchange rate, rounding, and price charm for the currency.
* Store settings are available, but they are not hooked to any additional functionality. 

Known issues:
* ~Styling is not complete: #1920 > PR: #1929~
* ~Search in modal is not added: #1924 > PR: #1971~
* Exchange rate is not listed in enabled currencies list: #1923 > PR: #1994
* ~Currency symbol displays html entity: #1922 > PR: #1930~
* ~Enabled currencies list does not refresh on update: #1921 > PR: #1956~
* ~Edit button also calls delete function to delete the currency from the list. > PR: #1929~
* ~If the automatic rate is chosen, the manual rate field should hide: #1925 > PR: ##1977~
* Preview on single currency page is not functional: https://github.com/Automattic/woocommerce-payments/issues/1928
* There are other TODOs in the code that may not be as large as these issues.
* Tests are not created.

#### Testing instructions

* Switch to using the branch.
* Navigate to WooCommerce > Settings Multi-Currency.
* Click the Add Currencies button, modify the checked currencies in the modal, and click to save.
* Refresh, and you will see the changes.
* Click the edit button for a currency, modify any settings, and then save.

-------------------
